certbot/ocsp.py

```diff
@@ -1,53 +1,80 @@
 """Tools for checking certificate revocation."""
 import logging
 import re
-
+from datetime import datetime, timedelta
 from subprocess import Popen, PIPE
 
+try:
+    # Only cryptography>=2.5 has ocsp module
+    # and signature_hash_algorithm attribute in OCSPResponse class
+    from cryptography.x509 import ocsp  # pylint: disable=import-error
+    from cryptography.x509.ocsp import OCSPResponse  # pylint: disable=import-error,unused-import
```

Comment:
nit: Can we remove this line and just use `ocsp.OCSPResponse` in the line below?

Reply:
Done.